ci/setup-build-test-gdk.ps1

```diff
@@ -70,25 +70,25 @@ $tests = @()
 
 if ((Test-Path env:TEST_CONFIG) -And ($env:TEST_CONFIG -eq "Native")) {
     # We run spatial tests against Vanilla UE4
-    $tests += [TestSuite]::new($gdk_test_project, "NetworkingMap", "VanillaTestResults", "/Game/Maps/FunctionalTests/CI_Fast/", "$user_gdk_settings", $False, "$user_cmd_line_args")
-    
+    $tests += [TestSuite]::new($gdk_test_project, "SpatialNetworkingMap", "VanillaTestResults", "/Game/Intermediate/Maps/CI_Premerge/", "$user_gdk_settings", $False, "$user_cmd_line_args")
+
     if ($env:SLOW_NETWORKING_TESTS -like "true") {
-        $tests[0].tests_path += "+/Game/Maps/FunctionalTests/CI_Slow/"
+        $tests[0].tests_path += "+/Game/Intermediate/Maps/CI_Nightly/"
         $tests[0].test_results_dir = "Slow" + $tests[0].test_results_dir
-        
+
         # And if slow, we run NetTest functional maps against Vanilla UE4 as well
         $tests += [TestSuite]::new($native_test_project, "NetworkingMap", "NativeNetTestResults", "/Game/NetworkingMap", "$user_gdk_settings", $False, "$user_cmd_line_args")
     }
 }
 else {
     # We run all tests and networked functional maps
```

Comment:
`/Game/Intermediate/Maps/` is reused a lot in this file, perhaps make into a const